gdx/src/com/badlogic/gdx/math/Intersector.java

```diff
@@ -499,111 +499,116 @@ public static boolean intersectRaySphere (Ray ray, Vector3 center, float radius,
 
 	/** Intersects a {@link Ray} and a {@link BoundingBox}, returning the intersection point in intersection. This intersection is
 	 * defined as the point on the ray closest to the origin which is within the specified bounds.
-	 * 
+	 * defined as the point where the ray hits the box.
```

Comment:
i think you can revert javadoc changes (both formatting and this statement), current definition is more accurate.